### PR TITLE
Shared GAE dispatch config

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Part one of centralizing dispatch rules. static dispatch.yaml file will live in ddp-angular repo.
Part two is getting it from pepper-apis. Coming real soon.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -154,7 +159,9 @@ commands:
           name: Update GAE dispatch config
           command: |
             set -u
-            gcloud app deploy --quiet --project "broad-ddp-${ENVIRONMENT}" "${ANGULAR_PROJECT_DIR_PATH}/output-config/dispatch.yaml"
+            DISPATCH_FILE_PATH="<< parameters.config_deploy_dir >>/${ENVIRONMENT}/dispatch.yaml"
+            gcloud app deploy --quiet --project "broad-ddp-${ENVIRONMENT}" "${DISPATCH_FILE_PATH}"
+          working_directory: ~/repo
       - run:
```
Comment (MocanaAtBroad) [inline .circleci/config.yml:165]: This is where the dispatch magic happens.

File: config/dispatch.yaml.ctmpl [hunk 2]
```diff
@@ -1,40 +0,0 @@
...
-{{else if eq $environment "prod"}}
-  - url: "ascproject.org/*"
-    service: angio
-  - url: "mbcproject.org/*"
-    service: mbc
-  - url: "braincancerproject.org/*"
-    service: brain
-  - url: "osproject.org/*"
-    service: osteo
-{{end}}
```
Comment (MocanaAtBroad) [inline config/dispatch.yaml.ctmpl:40]: No longer needed.

File: config/deployment/dev/dispatch.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    service: angio
+  - url: "mbc.dev.datadonationplatform.org/*"
+    service: mbc
+  - url: "brain.dev.datadonationplatform.org/*"
+    service: brain
+  - url: "osteo.dev.datadonationplatform.org/*"
+    service: osteo
+  - url: "pepper-backend.dev.datadonationplatform.org/*"
+    service: pepper-backend
+
```
Comment (MocanaAtBroad) [inline config/deployment/dev/dispatch.yaml:15]: It appears that GCP insists on the name of this file being dispatch.yaml . The command complains about the syntax of the file which led me to check again and again that I was not screwing up white spaces in the yaml. At any rate, there is one of these files for each environment.

File: config/deployment/staging/dispatch.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+dispatch:
```
Comment (zyme) [inline config/deployment/staging/dispatch.yaml:1]: Can you add:
```
  - url: "pepper-staging.datadonationplatform.org/*"
    service: pepper-backend
```

File: config/deployment/test/dispatch.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+dispatch:
+  - url: "angio.test.datadonationplatform.org/*"
+    service: angio
+  - url: "mbc.test.datadonationplatform.org/*"
+    service: mbc
+  - url: "brain.test.datadonationplatform.org/*"
+    service: brain
+  - url: "osteo.test.datadonationplatform.org/*"
+    service: osteo
+  - url: "pepper-backend.test.datadonationplatform.org/*"
```
Comment (zyme) [inline config/deployment/test/dispatch.yaml:10]: Can you add:
```
  - url: "pepper-test.datadonationplatform.org/*"
    service: pepper-backend
```

File: config/deployment/dev/dispatch.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+dispatch:
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: angio
+  - url: "mbc.dev.datadonationplatform.org/*"
+    service: mbc
+  - url: "brain.dev.datadonationplatform.org/*"
+    service: brain
+  - url: "osteo.dev.datadonationplatform.org/*"
+    service: osteo
+  - url: "pepper-backend.dev.datadonationplatform.org/*"
```
Comment (zyme) [inline config/deployment/dev/dispatch.yaml:11]: Can you add:
```
  - url: "pepper-dev.datadonationplatform.org/*"
    service: pepper-backend
```

File: config/deployment/prod/dispatch.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+dispatch:
+  - url: "ascproject.org/*"
+    service: angio
+  - url: "mbcproject.org/*"
+    service: mbc
+  - url: "braincancerproject.org/*"
+    service: brain
+  - url: "osproject.org/**"
+    service: osteo
+  - url: "pepper-backend.prod.datadonationplatform.org/*"
```
Comment (zyme) [inline config/deployment/prod/dispatch.yaml:10]: Can you add:

  - url: "pepper.datadonationplatform.org/*"
    service: pepper-backend

**Review by zyme — APPROVED** [review]

**Pull request merged**